### PR TITLE
LTS Strategy and Scanner GHA

### PR DESCRIPTION
**Pull request by swcurran**
With apologizes to @pradeepp88 -- I've taken the text from his PR #3051, reworked it and republished the changes in this PR. We really need to get this PR published and included in 1.0.0, and we are very anxious to get 1.0.0 out. I did try to submit the changes as a PR to your branch, but GitHub would not allow me to do so.

I've made fairly extensive updates to the text to match closely the process we are following.  The good news is that other than the extra (and needed) Snyk GHA, the "new" policy matches the practices of the project, so not much needs to change.

### CODE REVIEW

**Review by jamshale — APPROVED** [review]
Read through it and looks good to me :+1:

**Review by jamshale — DISMISSED** [review]

File: .github/workflows/snyk-lts.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# This workflow will scan and monitor all the LTS releases for vulnerabilities with Snyk 
+name: Snyk Container for LTS Releases
+on:
+  push:
+    # These branches represent the LTS releases
+    branches:
+      - 0.12.x
+      - 0.11.x
```
Comment (esune) [inline .github/workflows/snyk-lts.yml:8]: Should be `0.12.lts` and `0.11.lts` to match what the document prescribes

**Review by esune — CHANGES_REQUESTED** [review]

**Review by esune — APPROVED** [review]
LGTM 👍🏻

File: .github/workflows/snyk-lts.yml (hunk 1 above)
Comment (jamshale) [reply .github/workflows/snyk-lts.yml]: Wasn't sure if we decided to change the branch names or not. We'll have to update them in the repo as well.

**Review by dbluhm — APPROVED** [review]

**Pull request merged**